Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6550,6 +6550,19 @@ UseGPUProcessForMediaEnabled:
       "ENABLE(GPU_PROCESS_BY_DEFAULT)": true
       default: false
 
+UseGPUProcessForMockMediaSource:
+  type: bool
+  status: embedder
+  category: media
+  humanReadableName: "GPU Process: MockMediaSource"
+  humanReadableDescription: "Run MockMediaSource in the GPU process"
```

Comment:
Why do we need this option if we enable this through Internals API, which is available to WTR and API tests?
If you plan to be able to switch it on in MiniBrowser or Safari, should the status be internal?

Reply:
internal definition is `Tools for debugging the WebKit engine. Not generally useful to web developers.
#   OFF by default.` but this isn't going to be off by default for very long.

For now it's just to unable locally while I go through all the test failures and fixes them.
Once all bugs have been exposed (and fixed) I'll change the default to true.

Reply:
The internal call only enable the MockMediaSource ; it doesn't control if it is to run in the GPU Process or not; which is what that pref does.

Reply:
Discussed this offline and this flag should only be temporary since 'using mock' is done through internals and the MSE in GPUProcess is a different option.
Or we should rename this flag to MockMediaSourceEnabled if we want to keep it for longer and remove the internals API that allow using mock.